<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 2009 Nokia Corporation and/or its subsidiary(-ies).
** Contact: Qt Software Information (qt-info@nokia.com)
**
** This file is part of the Qt Designer of the Qt Toolkit.
**
** $QT_BEGIN_LICENSE:LGPL$
** Commercial Usage
** Licensees holding valid Qt Commercial licenses may use this file in
** accordance with the Qt Commercial License Agreement provided with the
** Software or, alternatively, in accordance with the terms contained in
** a written agreement between you and Nokia.
**
** GNU Lesser General Public License Usage
** Alternatively, this file may be used under the terms of the GNU Lesser
** General Public License version 2.1 as published by the Free Software
** Foundation and appearing in the file LICENSE.LGPL included in the
** packaging of this file.  Please review the following information to
** ensure the GNU Lesser General Public License version 2.1 requirements
** will be met: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html.
**
** In addition, as a special exception, Nokia gives you certain
** additional rights. These rights are described in the Nokia Qt LGPL
** Exception version 1.0, included in the file LGPL_EXCEPTION.txt in this
** package.
**
** GNU General Public License Usage
** Alternatively, this file may be used under the terms of the GNU
** General Public License version 3.0 as published by the Free Software
** Foundation and appearing in the file LICENSE.GPL included in the
** packaging of this file.  Please review the following information to
** ensure the GNU General Public License version 3.0 requirements will be
** met: http://www.gnu.org/copyleft/gpl.html.
**
** If you are unsure which license is appropriate for your use, please
** contact the sales department at qt-sales@nokia.com.
** $QT_END_LICENSE$
**
*********************************************************************</comment>
 <class>qdesigner_internal::ItemListEditor</class>
 <widget class="QWidget" name="qdesigner_internal::ItemListEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Items List</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsLayout" >
        <item>
         <widget class="QToolButton" name="newListItemButton" >
          <property name="toolTip" >
           <string>New Item</string>
          </property>
          <property name="text" >
           <string>&amp;New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteListItemButton" >
          <property name="toolTip" >
           <string>Delete Item</string>
          </property>
          <property name="text" >
           <string>&amp;Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>16</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="moveListItemUpButton" >
          <property name="toolTip" >
           <string>Move Item Up</string>
          </property>
          <property name="text" >
           <string>U</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="moveListItemDownButton" >
          <property name="toolTip" >
           <string>Move Item Down</string>
          </property>
          <property name="text" >
           <string>D</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="showPropertiesButton" >
          <property name="text" >
           <string>Properties &amp;>></string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
